data_prep/mp_prep_other_inputs_annual.py

```diff
@@ -38,140 +43,194 @@ def mp_prep_other_inputs(tile_id_list):
+
+    if process == 'tclf' or process == 'all':
+        # List of output directories and output file name patterns
+        output_dir_list = [
+                            cn.TCLF_processed_dir
+        ]
+
+        output_pattern_list = [
+                                cn.pattern_TCLF_processed
+        ]
+
+        # If the model run isn't the standard one, the output directory and file names are changed
+        if sensit_type != 'std':
+            uu.print_log('Changing output directory and file name pattern based on sensitivity analysis')
+            output_dir_list = uu.alter_dirs(sensit_type, output_dir_list)
+            output_pattern_list = uu.alter_patterns(sensit_type, output_pattern_list)
+
+        # A date can optionally be provided by the full model script or a run of this script.
+        # This replaces the date in constants_and_names.
+        # Only done if output upload is enabled.
+        if cn.RUN_DATE is not None and cn.NO_UPLOAD is False:
+            output_dir_list = uu.replace_output_dir_date(output_dir_list, cn.RUN_DATE)
+
+        ### Tree cover loss from fires processing
+        uu.print_log("STEP 2: Preprocess tree cover loss from fires")
+
+        # TCLF is downloaded to its own folder because it doesn't have a standardized file name pattern.
+        # This way, the entire contents of the TCLF folder can be worked on without mixing with other files.
+        TCLF_s3_dir = os.path.join(cn.docker_tile_dir, 'TCLF')
+        if os.path.exists(TCLF_s3_dir):
+            os.rmdir(TCLF_s3_dir)
+        os.mkdir(TCLF_s3_dir)
+        cmd = ['aws', 's3', 'cp', cn.TCLF_raw_dir, TCLF_s3_dir, '--request-payer', 'requester',
+               '--include', '*', '--exclude', 'tiles*', '--exclude', '*geojason', '--exclude', '*Store', '--recursive']
+        uu.log_subprocess_output_full(cmd)
+
+        # Creates global vrt of TCLF
+        uu.print_log("Creating vrt of TCLF...")
+        tclf_vrt = 'TCLF.vrt'
+        os.system(f'gdalbuildvrt -srcnodata 0 {tclf_vrt} {TCLF_s3_dir}/*.tif')
+        uu.print_log("  TCLF vrt created")
+
+        # Creates TCLF tiles
+        source_raster = tclf_vrt
+        out_pattern = cn.pattern_TCLF_processed
+        dt = 'Byte'
+        if cn.count == 96:
+            processes = 34  # 30 = 510 GB initial peak; 34=600 GB peak
+        else:
+            processes = int(cn.count/2)
+        uu.print_log(f'Creating TCLF tiles with {processes} processors...')
+        pool = multiprocessing.Pool(processes)
+        pool.map(partial(uu.mp_warp_to_Hansen, source_raster=source_raster, out_pattern=out_pattern, dt=dt), tile_id_list)
+        pool.close()
+        pool.join()
+
+        for output_pattern in [
+            cn.pattern_TCLF_processed
+        ]:
+
+            if cn.count == 96:
+                processes = 50  # 60 processors = >730 GB peak (for European natural forest forest removal rates); 50 = XXX GB peak
```

Comment:
Same about deleting the comment about the processors here (and elsewhere it seems copied from some other usage).

Reply:
Changes made in the following commit: b359551c7aa4405098e8f07ea7a582c6eddb963c